tokenLists/ethereum.tokenlist.json

```diff
@@ -276,6 +276,14 @@
             "name": "Rice Wallet",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/KyberNetwork/ks-assets/main/img/token/RICE.png"
+        },
+        {
+            "chainId": 1,
+            "address": "0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25",
+            "symbol": "SLP",
+            "name": "Smooth Love Potion",
+            "decimals": 0,
+            "logoURI": "https://assets.coingecko.com/coins/images/10366/small/SLP.png"
```

Comment:
This should be the github link, the same as you did above

Reply:
updated.